packages/edit-site/src/hooks/push-changes-to-global-styles/index.js

```diff
@@ -115,7 +118,7 @@ function useChangesToPush( name, attributes ) {
 					];
 				const value = presetAttributeValue
 					? `var:preset|${ STYLE_PATH_TO_CSS_VAR_INFIX[ presetAttributeKey ] }|${ presetAttributeValue }`
-					: get( attributes.style, path );
+					: getValueFromObjectPath( attributes.style, path );
```

Comment:
`path` is expected to be an array, and for `attributes.style` we're handling nullish values just in case:

https://github.com/WordPress/gutenberg/blob/ae5cc08fe0f352c5e0b914ca741a8433e18a4e6e/packages/edit-site/src/hooks/push-changes-to-global-styles/index.js#L109